cookbook/deployment/heroku.rst

```diff
@@ -132,6 +132,18 @@ variables, you can issue a single command to prepare your app for a deployment:
 
     $ heroku config:set SYMFONY_ENV=prod
 
+.. caution::
+
+    Be aware that dependencies from ``composer.json`` listed in ``require-dev``
+    are never installed during a deploy on Heroku. This may cause problems if your
+    Symfony environment relies on such packages. For instance, the ``dev``
+    environment of the Symfony Standard Edition loads the SensioGeneratorBundle
```

Comment:
True. I was running into this when I first tried to deploy Symfony to Heroku as there was no best practice to set `SYMFONY_ENV`=`prod` and the solution for solving this issue was to move the SensioGeneratorBundle to Composer's  `require` list. No wondering if it makes sense to deploy `dev` ;-) So this caution note may clarifies why things do not work in `dev` if someone forgets to set `SYMFONY_ENV`=`prod`.

On the other hand, if you need some node packages for the `prod` build (e.g. to compile assets) you have to move them to the `dependencies` list of `packages.json` because `devDependencies` are not installed. Weird.